launch_crab_multithreading.py

```diff
@@ -49,7 +49,8 @@
 #enable multi-threading
 #remove from being default!
 #config.JobType.maxMemoryMB = 3000#15900 #more memory
-config.JobType.numCores = 8
+#deactivate
```

Comment:
I guess this is needed for AOD but can be kept otherwise?

Reply:
@lbenato, could you confirm whether this would be a good solution? Or should we merge the PR as is, and @MelanieEich and I then modify our lists to run on multiple cores as needed?

Reply:
Yes I am fine with your proposal, sorry I missed the comment

Reply:
So, just to make sure I understand correctly: We should expect one more commit with the number of cores modified only for calo lists, right?

Reply:
I think you can either comment it or put it differently. This number must be synchronized with the configuration file hence I will have to modify it anyways when submitting. It's just one line and you can comment it, no need to commit only this from my side

Reply:
Alright, then we can merge the PR as is. Thanks!